src/pages/register-a-beacon/land-other-communications.tsx

```diff
@@ -53,8 +52,7 @@ const definePageForm = ({
       [
         {
           dependsOn: "portableVhfRadio",
-          meetingCondition: (value) =>
-            value === Communication.PORTABLE_VHF_RADIO,
+          meetingCondition: (value) => value === "true",
```

Comment:
Changed these values to `true` over the `Communication` enum values as they are not needed.  The key is `portableVhfRadio` and the value will be `true` if the user has selected the checkbox.

Reply:
This makes more sense to me vs previously where `portableVhfRadio === "PORTABLE_VHF_RADIO"` to signify the user has one.  Boolean much clearer 👍